TicketsDemo.Data/OptionsForCalculationPrice/TeaCoffeeBedParametrs.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TicketsDemo.Data.Entities;
+
+namespace TicketsDemo.Data.OptionsForCalculationPrice
```

Comment:
shall not be a part of 'data' assembly, as it is not a persistent entity but DTO needed by particular domain service, move it to domain assembly nearby to service, which use this dto